### ISSUE TITLE
Proxy redirects to ogiginal site which is blocked

### ISSUE THREAD

**Issue by CrateTech** [nfriedly/node-unblocker#188]
Whenever I go to certain sites such as armor games or yohoho.io, these sites just redirect to their original link and d not use the proxy. THese websites are blocked so it makes them unusable.

**Comment by SyrusPlayz** [comment]
yeah it does the same thing for me

**Comment by nfriedly** [comment]
You could try out the v3 branch - https://github.com/nfriedly/node-unblocker/tree/v3 - it is a little bit more aggressive about rewriting client-side scripts, but it breaks enough things that it isn't really ready for prime-time just yet.

**Comment by CrateTech** [comment]
How do I install v3 using npm? It looks to have the same command as the old one, so IU think its installing the old one.

**Comment by CrateTech** [comment]
I forgot to mention, I use this to run it https://github.com/nfriedly/nodeunblocker.com so how do I update that to v3?

**Comment by nfriedly** [comment]
In the `nodeunblocker.com` folder, run:

`npm install git://github.com/nfriedly/node-unblocker.git#v3 --save`

Or, if you're deploying it somewhere like heroku, you'll have to make a fork and edit the [package.json](https://github.com/nfriedly/nodeunblocker.com/blob/711eebc5824205a77b7a2fa15857af7c4c82cee1/package.json#L13) to use that `git://...` url instead of `^2.2.0`

**Comment by CrateTech** [comment]
yeah, unfortunately both the sites i mentioned still redirect, and the site that gats the most traffic breaks, so Ill just stick with the n0rmalk one for now, but I will follow updates on v3. thank you for your work and time.
